Symfony/CS/AbstractAlignFixer.php

```diff
@@ -86,4 +91,21 @@ protected function replacePlaceholder(Tokens $tokens, $deepestLevel)
 
         return $tmpCode;
     }
+
+    /**
+     * Cleans up extra line between variable and placeholder.
+     * Ex: `$ccc  = 1` becomes `$ccc = 1`.
+     *
+     * @param string $line
+     * @param string $placeholder
+     *
+     * @return string
+     */
+    private function cleanInnerSpaces($line, $placeholder)
```

Comment:
instead od putting spaces before `=` we first trim spaces and then put less spaces.
maybe it is a quick solution but this will make this fixer slower :/